.github/workflows/schedule-update-actions.yml

```diff
@@ -0,0 +1,26 @@
+# SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: GitHub Actions Version Updater
+on:
+  workflow_dispatch:
+  schedule:
+    # Automatically run on every Sunday
+    - cron:  '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3.5.2
+        with:
+          # [Required] Access token with `workflow` scope.
+          token: ${{ secrets.PAT }}
```

Comment:
I don't think actions/checkout should need any tokens?
It's already used e.g. here: https://github.com/tiiuae/ghaf-infra/blob/main/.github/workflows/test-ghaf-infra.yml#L18 with no token.

Reply:
Great point, I'll test this btw based on their docs  Access token was required..